roboteam_ai/src/skills/Attack.cpp

```diff
@@ -18,17 +25,12 @@ Attack::Attack(string name, bt::Blackboard::Ptr blackboard)
 }
 
 void Attack::onInitialize() {
-    shot = false;
-    shotControl = std::make_shared<control::ShotController>();
+    shotControl = std::make_shared<control::ShotController>(control::ShotPrecision::LOW, control::BallSpeed::MAX_SPEED, true);
```

Comment:
does not need to be a pointer but >> it's okay << what rolf would say

Reply:
Nice